cairo/kakarot-ssj/crates/utils/src/math.cairo

```diff
@@ -367,6 +370,7 @@ mod tests {
 
     #[test]
     fn test_wrapping_pow() {
+        assert(0_u256.wrapping_pow(0) == 1, '0^0 should be 1');
```

Comment:
Why not using the assert! macro ?

Reply:
No reason I did as the otherlines in the test. It could be moved to use assert ea